src/labels/CustomLabels.labels

```diff
@@ -6654,6 +6654,22 @@ Selecting Don't Auto Schedule Default NPSP Jobs tells NPSP not to check for or r
         <shortDescription>stgErrorInvalidNameFormat</shortDescription>
         <value>Error in {0}: {1}</value>
     </labels>
+    <labels>
+        <fullName>stgGauWithPaymentEnabledError</fullName>
```

Comment:
To be more consistent with error related labels, I recommend the following for the label names:

- stgErrorMissingDefaultGauRecord
- stgErrorMissingDefaultGauRecordWithPmtAllocEnabled